src/views/router_views/individual_statement/individual_statement.njk

```diff
@@ -0,0 +1,31 @@
+{% extends "layouts/default.njk" %}
+
+{% from "govuk/components/checkboxes/macro.njk" import govukCheckboxes %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+
```

Comment:
I think we need `{% set title = i18n.individual_statement_title %}` in order for the browser tab to display the title.
(`title` gets used in `partials/__meta_header.njk`

Reply:
I don't see this added yet

Reply:
Check now :)

Reply:
👍